spec/defines/command_spec.rb

```diff
@@ -19,5 +19,9 @@
   end
 
   it { is_expected.to compile.with_all_deps }
-  it { is_expected.to contain_file('/etc/nagios/nrpe.d/check_users.cfg') }
+  it {
+    is_expected.to contain_file('/etc/nagios/nrpe.d/check_users.cfg').with(
```

Comment:
I often prefer `with_mode('0644')` but I don't think there's actually a difference.

Reply:
I figured I'd probably be adding checks for the other parameters before too long.